css-three-column-layout/styles.css

```diff
@@ -0,0 +1,94 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+img {
+  width: 50%;
+  height: 50%;
+  justify-content: center;
+  align-items: center;
+  margin-top: 25%;
+  margin-left: 25%;
+}
+
+.circle-border {
+  display: flex;
+  border: solid;
+  border-color: white;
+  border-radius: 50%;
+  width: 100px;
+  height: 100px;
+  background-color: white;
+  margin-left: 30%;
+}
+
+body {
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  src: url("https://fonts.gstatic.com/s/lato/v15/S6uyw4BMUTPHjx4wXiWtFCc.woff2") format("woff2");
+  unicode-range:
+    u+0000-00ff,
+    u+0131,
+    u+0152-0153,
+    u+02bb-02bc,
+    u+02c6,
+    u+02da,
+    u+02dc,
+    u+2000-206f,
+    u+2074,
+    u+20ac,
+    u+2122,
+    u+2191,
+    u+2193,
+    u+2212,
+    u+2215,
+    u+feff,
+    u+fffd;
```

Comment:
These are only necessary in the `@font-face` rule. In order to use a font family declared in a font face rule, you only need the `font-family` property